README.md

```diff
@@ -515,10 +515,21 @@ Options:
   Default is `false.`
 * `dangerousExtend`: Disable security checks for `extend` query.
   Default is `false.`
-* `mobileToDesktop`: Use desktop browsers if Can I Use doesn’t have data
-  about this mobile version. For instance, Browserslist will return
-  `chrome 20` on `and_chr 20` query (Can I Use has only data only about
-  latest versions of mobile browsers). Default is `false`.
+* `mobileToDesktop`: Resolve the mobile version using the desktop version when
+  Can I Use doesn't have data about the specified version. For instance, it
+  will return two Chrome for Android versions even if Can I Use has only
+  data only about the latest version. Default is `false`.
+* `normalizers`: An array of names of normalizers below and functions
+  that accept a browser and return the normalized versions, which are
+  applied to resolved browsers in order. For instance, `['toChromium', 'toDesktop']`.
+  * `toChromium`: Normalize Chromium-based browsers to Chrome. For instance,
+    UC Browser, QQ Browser, Baidu browser for Android, and Samsung Internet
+    will return `and_chr` with the version of Chromium they are based on.
+    Note Edge and Opera is not normalized. Gecko-based browsers are
+    also normalized to Firefox, e.g., KaiOS Browser will return `and_ff`.
+  * `toDesktop`: Normalize mobile browsers to desktop browsers.
```

Comment:
I've named it `toDesktop` instead of `mobileToDesktop` as it may be confused with the existing option.

Reply:
What is the difference between `normalizers: ['toDesktop']` and `mobileToDesktop: true`?

Reply:
@ai See #510 and https://github.com/browserslist/browserslist/pull/511/commits/2ca9e82f71868a49473a80c2702b3b88793d9621. `mobileToDesktop` only resolves the version part using desktop data. For example:
```bash
$ node cli.js "last 5 and_chr versions"
and_chr 81
$ node cli.js --mobile-to-desktop "last 5 and_chr versions"
and_chr 83
and_chr 81
and_chr 80
and_chr 79
and_chr 78
$ node cli.js --normalizers=toDesktop "last 5 and_chr versions"
chrome 81
$ node cli.js --normalizers=toDesktop --mobile-to-desktop "last 5 and_chr versions"
chrome 83
chrome 81
chrome 80
chrome 79
chrome 78

```

Reply:
@JLHwung what do you think if we change `mobileToDesktop` behavior? On `last 5 and_chr versions` we will return `chrome 78-83` instead of `and_chr 78-83`. Will it affect Babel?

Reply:
Internally Babel maps `and_chr` to `chrome` and it seems to me this PR moves this process to `browserslists`. I don't think it will affect Babel.

Another thoughts about `mobileToDesktop`: AFAIK opera_mob is mapped to opera, this is incorrect because they are if not always, frequently based on different chromium versions. Can we incorporate data from https://github.com/mdn/browser-compat-data/blob/master/browsers/opera_android.json ? This can help `preset-env` on supporting `opera_mob`. Surely we can address it in a different PR.

Reply:
I've added `'op_mob 46': 63` to `browserslist.normalizedVersions` (`byEngine`). I think removing it from `desktopNames` (`mobileToDesktop`) would be out of scope for this PR.